src/fsharp/Fsc/fsc.exe.config

```diff
@@ -5,13 +5,12 @@
     <gcServer enabled="true"/>
     <assemblyBinding xmlns="urn:schemas-microsoft-com:asm.v1">
       <dependentAssembly>
-        <assemblyIdentity
-          name="FSharp.Core"
-          publicKeyToken="b03f5f7f11d50a3a"
-          culture="neutral"/>
-        <bindingRedirect
-          oldVersion="2.0.0.0-4.4.1.0"
-          newVersion="4.4.1.0"/>
+        <assemblyIdentity name="FSharp.Core" publicKeyToken="b03f5f7f11d50a3a" culture="neutral"/>
+        <bindingRedirect oldVersion="2.0.0.0-4.4.1.0" newVersion="4.4.1.0"/>
+      </dependentAssembly>
+      <dependentAssembly>
+        <assemblyIdentity name="System.Collections.Immutable" publicKeyToken="b03f5f7f11d50a3a" culture="neutral" />
+        <bindingRedirect oldVersion="0.0.0.0-1.2.1.0" newVersion="1.2.1.0" />
```

Comment:
Just a reminder that when using nuget packages, paket actually does a good work at generating those redirects (related to nuget packages you are pulling) for you.

I've spent too much time with conflicts and binding redirects, msbuild or nuget are just hopeless in helping the developer do the right thing in this context.

When I switched to paket at my work, it simply relieved me from that problem, I don't face binding redirects issues, or only in case I'm pulling dependencies outside of nuget eco system (which is very seldom).

This is the only fsharp repository which is not using paket so maybe we could do some harmonization?